types/speakeasy/index.d.ts

```diff
@@ -1,9 +1,17 @@
 // Type definitions for speakeasy v2.0.0
-// Project: https://github.com/markbao/speakeasy
-// Definitions by: Lucas Woo <https://github.com/legendecas>, Alexander Batukhtin <https://github.com/mrOlorin>
+// Project: https://github.com/speakeasyjs/speakeasy
+// Definitions by: Lucas Woo <https://github.com/legendecas>, Alexander Batukhtin <https://github.com/mrOlorin>, Aayush Kapoor <https://github.com/xeoneux>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-interface Key {
+declare module "speakeasy" {
```

Comment:
Do not use `declare module "speakeasy"` instead use `export as namespace speakeasy`